hr_data_analytics_package_folder/params.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+## Global project params
+
+##################  CONSTANTS  #####################
+DATA_HR = os.environ.get("DATA_HR")
+DATA_HR_API = os.environ.get("DATA_HR_API")
```

Comment:
Due to name conflict on Google Artifact Registry, I had to change our folder name from hyphens (-) to underscores ( _ ).
This params.py file is considered new but same logic as previous version.